srcpkgs/wren/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'wren'
+pkgname=wren
+version=0.3.0
+revision=1
+build_wrksrc=./projects/make
+build_style=gnu-makefile
+#make_build_args=""
+#make_dirs="/var/log/dir 0755 root root"
+short_desc="Small, class-based, concurrent scripting language - library files"
+maintainer="cinerea0 <cinerea0@protonmail.com>"
+license="MIT"
+homepage="http://wren.io"
+distfiles="https://github.com/wren-lang/wren/archive/${version}.tar.gz"
+checksum=c566422b52a18693f57b15ae4c9459604e426ea64eddb5fbf2844d8781aa4eb7
+
+do_install() {
+	vinstall ./lib/libwren.a 0755 /usr/lib/
+	vinstall ./lib/libwren.so 0755 /usr/lib/
+	vlicense ./LICENSE
+}
+
+wren-devel_package() {
+	short_desc="Small, class-based, concurrent scripting language - source files"
+	depends="${sourcepkg}-${version}_${revision}"
+	pkg_install() {
+		vmkdir /usr/include/wren
+		vinstall "./src/include/wren.h" 0644 /usr/include/wren
+		vinstall "./src/optional/*" 0644 /usr/include/wren
+		vinstall "./src/vm/*" 0644 /usr/include/wren
+	}
+}
```

Comment:
Ideally the package generates `.so.*` files that should be installed in the package, while the `.so` and `.a` files should go in `-devel`. If it doesn't create those, keep `.so` files in the package, but `.a` should still go in the devel package.

Reply:
The package does not version the SO. This should probably be upstreamed so we can track disparate version numbers in future updates. This is important because, if any package depends on `libwren.so`, we wouldn't know whether it is necessary to rebuild those packages when you update `wren`.

Reply:
I've changed it so that the `.a` file goes in the devel package. As for versioning, should I open an issue in the original repository to ask about it?

Reply:
I think that would be advisable. SO versioning is how you indicate that the ABI has broken between releases.

Reply:
I wonder if SO versioning doesn't make sense for them because wren will be included inside other applications. Might make more sense to package wren-cli for now, and wren if we find an application that needs it.

Reply:
I second that.